### PR TITLE
Fix/multi account kinesis

### PR DESCRIPTION
**Pull request by macnev2013**
## Motivation
This PR fixes multi-account support for kinesis related tests. This is a follow-up PR for https://github.com/localstack/localstack/pull/9785. We have discovered that some of the changes made in https://github.com/localstack/localstack/pull/8204 were causing failure in test cases when running against [formatted-access-key-id-for-tests](https://github.com/localstack/localstack/tree/formatted-access-key-id-for-tests) branch.

## Changes
We have added `AWS_SECRET_ACCESS_KEY` additionally in the environment variable while running the KCL command. Since in the [original PR ](https://github.com/localstack/localstack/pull/9785) we have removed the function setting up access key and secret key globally.

## Testing
Added non-default account related changes in commit [f6c3fd](https://github.com/localstack/localstack/commit/a67f1f8520b999a0eb438aae73b3cda634809c3b).

## TODO

What's left to do:

- [ ] Let the CI run complete
- [ ] Revert the changes

### CODE REVIEW

**Review by thrau — COMMENTED** [review]
LGTM, just also tagging @dfangl to take a look, since the config/constants changes seem significant.

**Comment by macnev2013** [comment]
Hey @thrau, Apologies for the confusion. [This](https://github.com/localstack/localstack/pull/10188/commits/a67f1f8520b999a0eb438aae73b3cda634809c3b) commit will be reverted.

File: localstack/utils/kinesis/kinesis_connector.py [hunk 1]
```diff
@@ -245,6 +245,7 @@ def _start_kcl_client_process(
     env_vars = {
         "AWS_CBOR_DISABLE": "true",
         "AWS_ACCESS_KEY_ID": account_id,
+        "AWS_SECRET_ACCESS_KEY": account_id,
```
Comment (dfangl) [inline localstack/utils/kinesis/kinesis_connector.py:248]: I think it is fine to set this here, but for this value it should be possible to set anything. Any particular reason to set it to the account id?

**Review by dfangl — APPROVED** [review]
Looks good :)

File: localstack/utils/kinesis/kinesis_connector.py (hunk 1 above)
Comment (macnev2013) [reply localstack/utils/kinesis/kinesis_connector.py:248]: For `AWS_SECRET_ACCESS_KEY`, we can set `internal secret key (__internal__)` or `account_id` here. @viren-nadkarni suggested to use account id here.

**Comment by macnev2013** [comment]
Ext Integration Tests workflow run: https://github.com/localstack/localstack-ext/actions/runs/7812160849

**Comment by macnev2013** [comment]
> Ext Integration Tests workflow run: https://github.com/localstack/localstack-ext/actions/runs/7813257205

Ext pipeline is 🟢

**Pull request merged**